io/io/test/CMakeLists.txt

```diff
@@ -18,6 +18,8 @@ if(uring AND NOT DEFINED ENV{ROOTTEST_IGNORE_URING})
   ROOT_ADD_GTEST(RIoUring RIoUring.cxx LIBRARIES RIO)
 endif()
 
+ROOT_ADD_GTEST(rfile rfile.cxx LIBRARIES RIO Hist ROOTNTuple)
```

Comment:
In principle, `ROOTNTuple` is a higher level in terms of layering than `RIO`, not sure if it's a good idea to depend on it (even for a test)...

Reply:
Isn't that the case also for `Hist`?

Reply:
Those test should probably be moved to roottest